source/NVDAObjects/UIA/wordDocument.py

```diff
@@ -483,6 +490,15 @@ def _getFormatFieldAtRange(self, textRange, formatConfig, ignoreMixedValues=Fals
 					)
 					if isinstance(textColumnNumber, int):
 						formatField.field["text-column-number"] = textColumnNumber
+		expandCollapseState = UIARemote.msWord_getCustomAttributeValue(
+			docElement,
+			textRange,
+			UIACustomAttributeID.EXPAND_COLLAPSE_STATE,
+		)
+		if expandCollapseState == EXPAND_COLLAPSE_STATE.COLLAPSED:
+			formatField.field["collapsed"] = True
+		elif expandCollapseState == EXPAND_COLLAPSE_STATE.EXPANDED:
+			formatField.field["collapsed"] = False
```

Comment:
I get the following error with an older Word version:
`UnboundLocalError: cannot access local variable 'docElement' where it is not associated with a value`

You should indent this code to execute it in the "if" branch where `docElement` is defined.

```suggestion
			expandCollapseState = UIARemote.msWord_getCustomAttributeValue(
				docElement,
				textRange,
				UIACustomAttributeID.EXPAND_COLLAPSE_STATE,
			)
			if expandCollapseState == EXPAND_COLLAPSE_STATE.COLLAPSED:
				formatField.field["collapsed"] = True
			elif expandCollapseState == EXPAND_COLLAPSE_STATE.EXPANDED:
				formatField.field["collapsed"] = False
```